### ISSUE TITLE
Troubles with ./ in make

### ISSUE THREAD

**Issue by zxqfox** [enb-bem/enb-magic-platform#35]
How to repeat:
```
magic make ./path
```

Expected:
```
22:20:53.039 - build started
22:20:53.383 - [rebuild] [...] file-provider
...
etc.
```

Actual:
```
22:24:21.900 - build started
22:24:22.188 - build finished - 1638ms
22:24:22.189 - build finished - 1639ms
```

**Comment by andrewblond** [comment]
Fixed in `enb-magic-factory@0.4.2`.

**Comment by zxqfox** [comment]
Looks like it's actual again.
